app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/HomePresenter.kt

```diff
@@ -0,0 +1,52 @@
+package com.sun_asterisk.moviedb_50.screen.home
+
+import android.util.Log
+import com.sun_asterisk.moviedb_50.data.model.Genres
+import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.data.source.remote.OnFetchDataJsonListener
+import com.sun_asterisk.moviedb_50.data.source.remote.Repository
+
+
+class HomePresenter(private val mRepository: Repository?) :
+    HomeContract.Presenter {
+    private var view: HomeContract.View? = null
+    override fun onStart() {
```

Comment:
```suggestion
    private var view: HomeContract.View? = null
    
    override fun onStart() {
```
function thì cách ra 1 dòng